src/utils/RestfulUtils.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import { ControllerError } from "../enums/ControllerError";
+import { StatusCode } from "../enums/StatusCode";
+import { IRestfulObject } from "../types/BaseResponseApp";
+
+// Utils used by Controllers
+export class RestfulUtils {
+  // Check if token is set into request. If token is invalid and res is provided, an error will be written
+  public static checkTokenIntoRequest(
+    req: express.Request,
+    res?: express.Response
+  ): boolean {
+    if (req.query.token === undefined) {
+      if (res !== undefined) {
+        this.setErrorResponse(res, ControllerError.ERROR_INVALID_TOKEN);
+      }
+      return false;
+    }
+    return true;
+  }
+
+  // Set an error message for express response
+  public static setErrorResponse(
+    res: express.Response,
+    errorMsg: string
+  ): void {
+    console.error("Controller response is an error: " + errorMsg);
+    res.status(500).json({ status: StatusCode.ERROR, errorMessage: errorMsg });
+  }
+
+  // Set a success message for express response
+  public static setSuccessResponse(
+    res: express.Response,
+    content: IRestfulObject | Error
+  ): void {
+    if (content instanceof Error) {
+      console.error("Controller response is an error: " + content.message);
+      res.status(200).json({
+        status: StatusCode.ERROR,
```

Comment:
is it's an error why it returns 200 ?

Reply:
Changed. It will be available into next Push